README.md

```diff
@@ -7,6 +7,24 @@ State-of-the-art models in AI,ML,NLP & Vision for FinTech community by Yubi Data
 * Models files hosted on FTP server and are downloaded when constructor is called
 * Curently tested on `Ubuntu 20.04`, `MacOS 12.3` and `python >=3.7`
 
+## How to install package
+* Clone the git repository or download zip/tar files and unzip
+```
+cd /parent/directory/path/of/YubiAI/
+```
+* Install only NLP dependencies
+```
+pip install ".[nlp]"
+```
+* Install only Vision dependencies
+```
+pip install ".[cv]"
+```
+* Install both NLP & Vision dependencies
+```
+pip install ".[nlp,cv]"
+```
+
 ## How to use package
```

Comment:
You need to change this as well. With and without installing pkg codes. The following codes won't work.

Reply:
You also need to change all readme pkg links in `Whats New in NLP and CV` part .. they also won't work.

Reply:
Check .. all possible locations where links or classes are called .. all of them have to be corrected

Reply:
Done.